spec/nylas/http_client_spec.rb

```diff
@@ -43,6 +43,23 @@
     end
   end
 
+  describe "#execute" do
+    it "includes Nylas API Version in headers" do
+      nylas = described_class.new(app_id: "id", app_secret: "secret", access_token: "token")
+      allow(RestClient::Request).to receive(:execute)
+
+      nylas.execute(method: :get, path: "/contacts/1234/picture")
+
+      expect(RestClient::Request).to have_received(:execute).with(
+        headers: hash_including("Nylas-API-Version" => "2.2"),
```

Comment:
Consider replacing "2.2" with a reference to the constant in the class under test - probably will be less brittle.